packages/cmr-client/src/CMR.ts

```diff
@@ -31,12 +30,10 @@ async function updateToken(
   username: string,
   password: string
 ): Promise<string> {
-  const earthdataLoginObject = new EarthdataLogin({
-    username: username,
-    password: password,
-    edlEnv: process.env.CMR_ENVIRONMENT,
-  });
-  return await earthdataLoginObject.getEDLToken();
+  // eslint-disable-next-line @typescript-eslint/dot-notation
+  const edlEnv = process.env['CMR_ENVIRONMENT'];
+  if (!edlEnv) throw new Error('CMR_ENVIRONMENT not set');
```

Comment:
You can use the `getRequiredEnvVar` function from `packages/common/src/env.ts` here.

Reply:
`getRequiredEnvVar('CMR_ENVIRONMENT')`